CHANGELOG.md

```diff
@@ -49,10 +49,10 @@ Relax the restrictions on domain validation so that we allow unicode domains and
 other non ASCII domains while still disallowing the domains we blocked before.
 
 ## Version 3.0.0
-* Moved __and__ renamed blacklist and whitelist and disposable_emails. Moved from the vendor directory to
+* Moved __and__ renamed blocklist and allowlist and disposable_emails. Moved from the vendor directory to
   the config directory.  
-  `vendor/blacklist.yml` -> `config/blacklisted_email_domains.yml`  
-  `vendor/whitelist.yml` -> `config/whitelisted_email_domains.yml`  
+  `vendor/blocklist.yml` -> `config/blocklisted_email_domains.yml`  
+  `vendor/allowlist.yml` -> `config/allowlisted_email_domains.yml`  
```

Comment:
This shouldn't be changed as this is the changelog. At v3, these were the actual paths.